pages/dashboard.vue

```diff
@@ -31,6 +31,22 @@
               Administrator of your BC Registries account.
             </p>
           </v-container>
+          <a
+          class="text-decoration-none" target="_blank" rel="noopener noreferrer"
+          :href="'https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/'
+            +'permits-licences/news-updates/modernization-updates/using-the-business-registry'">
+            <v-container fluid rounded class="dash-container-info service-container mt-5 white">
+              <h4>Help with How to Use BC Registries Products and Services</h4>
+              <p class="ma-0 pt-3">
+                Learn about the registries and how to use the applications through step-by-step guides,
+                 online videos, and downloadable quick guides.
```

Comment:
You have additional spacing here

Reply:
oh I didn't know newline is treated as a space! fixed